module/input/Camera/src/Camera.cpp

```diff
@@ -66,13 +66,27 @@ namespace input {
         	}
         });
 
-        // When we shutdown, we must tell our camera class to close (stop streaming)
-        on<Shutdown>().then([this] {
+            on<Shutdown>().then([this] {
```

Comment:
You should install the clang-format plugin so the indentation stops breaking everywhere